frontend/src/components/common/form/NotesModal.tsx

```diff
@@ -8,19 +8,24 @@ import { useState } from 'react';
 import { FaRegFileAlt } from 'react-icons/fa';
 import { withNameSpace } from 'utils/formUtils';
 
-export interface IAddLeaseTenantNoteProps {
+export interface INotesModalProps {
+  notesLabel: React.ReactNode;
+  title: string;
   nameSpace?: string;
+  onSave?: (values: any) => void;
```

Comment:
I like making this a generic notes modal. One suggestion - could you please make the field name configurable via props? Right now is hardcoded to `notes` and it might not fit all forms